bluecode-pay/src/main/java/com/example/bluecodepay/bluecode/client/BluecodeClientFallback.java

```diff
@@ -5,6 +5,7 @@
 import com.example.bluecodepay.bluecode.response.BluecodeResponse;
```

Comment:
This comment regards to the maven.
1) Replace hibernate-core dependency inside blucode module, template module with javax.persistence-api
2) Make your parent module inherit spring boot parent. Inside your parent module use dependency management tag. https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#dependency-management
I recommend to read all this article but pay special attention to dependency-management part. 
It'll allow us to properly handle versions of our dependencies. 
3) Move versions of your dependencies to properties.  https://maven.apache.org/pom.html#properties